admin/modules/user/users.php

```diff
@@ -992,7 +992,7 @@
 	}
 	else
 	{
-		$user['avatar'] = "../".$mybb->settings['useravatar'];
+		$user['avatar'] = "../".str_replace('{theme}', 'images', $mybb->settings['useravatar']);
```

Comment:
{theme} here isn't actually the theme, it's `images`. I'm sure that isn't intended as per title